Python/pytime.c

```diff
@@ -815,6 +815,8 @@ pytime_as_timeval_struct(PyTime_t t, struct timeval *tv,
 #ifdef MS_WINDOWS
     // On Windows, timeval.tv_sec type is long
     res2 = _PyTime_AsCLong(tv_sec, &tv->tv_sec);
+#elif _UCRT
+    res2 = _PyTime_AsTime_t(tv_sec, (time_t *)&tv->tv_sec);
```

Comment:
This is not the correct fix. From the error message "expected 'time_t *' {aka 'long long int *'} but argument is of type 'long int *'" it can be seen that `tv->tv_sec` has type `long int` which is different from `long long int`.

Reply:
Actually my initial thought was to start with `struct timeval`, but maybe that's not appropriate :(
(If approved, I will edit it.)